config/environment.rb

```diff
@@ -1,12 +1,5 @@
 # Load the rails application
 require File.expand_path('../application', __FILE__)
 
-#Add timestamps and severity to environment.log
-class Logger
-  def format_message(severity, timestamp, progname, msg)
-    "#{timestamp.strftime('%Y-%m-%d %H:%M:%S')} [#{severity[0]}] #{msg}\n"
-  end
-end
```

Comment:
If you look at https://github.com/theforeman/foreman/pull/2376/files#diff-d71db24719150ef52a8afd6ef934f9b4R36 you'll see that I kept this same format but added in the name of the logger just before the severity. e.g.

```
2015-05-13 13:28:22 [app] [I] Connecting to database specified by database.yml
The Apipie cache is turned off. Enable it and run apipie:cache rake task to speed up API calls.
2015-05-13 13:28:33 [sql] [D]   Setting::General Load (0.9ms)  SELECT "settings".* FROM "settings" WHERE "settings"."category" IN ('Setting::General') AND "settings"."name" = 'fix_db_cache' ORDER BY name LIMIT 1
```